### PR TITLE
Update presence-admins.md

### PR DESCRIPTION
**Pull request by Reeced40**
https://github.com/MicrosoftDocs/OfficeDocs-SkypeForBusiness/issues/7199
Fixed sentence.

### CODE REVIEW

**Comment by scanum** [comment]
@JohanFreelancer9 Copy editing is needed for this PR. Thanks.

File: Teams/presence-admins.md [hunk 1]
```diff
@@ -47,7 +47,7 @@ Presence is part of a user's profile in Microsoft Teams (and throughout Microsof
 || ![Purple circle with arrow, indicates Out of office](media/Presence_OOF.png) Out of Office. Out of Office is used when an automatic reply is set. (Available in Outlook only.) |
 |||
  > [!Note]
```
Comment (JohanFreelancer9) [inline Teams/presence-admins.md:49]:
```suggestion
 > [!NOTE]
```

File: Teams/presence-admins.md [hunk 2]
```diff
@@ -47,7 +47,7 @@ Presence is part of a user's profile in Microsoft Teams (and throughout Microsof
...
- > For users that there mailbox is hosted onprem it is expected to have presence delays with a maximun of an hour.
+ > For users that have their mailbox hosted onprem, presence delays of one hour (maximum) are expected.
```
Comment (JohanFreelancer9) [inline Teams/presence-admins.md:50]:
```suggestion
 > For users that have their mailbox hosted on-prem, presence delays of one hour (maximum) are expected.
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review.

**Review by JohanFreelancer9 — APPROVED** [review]

**Comment by scanum** [comment]
@msdmaguire PR has been copyedited and is ready for final review, could you please check and merge? Thanks!

**Review by cazawideh — APPROVED** [review]

**Pull request merged**